data/json/items/gun/shot.json

```
@@ -845,5 +845,27 @@
     "modes": [ [ "DEFAULT", "double", 2 ] ],
     "//": "no provision for separate triggers",
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 2 } } ]
+  },
+  {
+    "id": "m26_mass_standalone",
+    "copy-from": "shotgun_base",
+    "type": "GUN",
+    "name": { "str": "M26-MASS standalone shotgun" },
+    "description": "Modular Accessory Shotgun System, an underbarrel bolt-action shotgun, inspired by \"Masterkey\", and fixed most of his problems.  This one was configured for a standalone usage.",
```

```suggestion
    "description": "Modular Accessory Shotgun System, an underbarrel bolt-action shotgun, inspired by \"Masterkey\", and fixed most of its problems.  This one was configured for a standalone usage.",
```
You should probably add what problems it fixed, as this is rather vague 😛